content/sensu-go/5.3/installation/auth.md

```diff
@@ -9,29 +9,30 @@ menu:
     parent: installation
 ---
 
-- [Managing authentication providers](#managing-authentication-providers) (enterprise-only)
-- [Configuring authentication providers](#configuring-authentication-providers) (enterprise-only)
-- [LDAP authentication](#ldap-authentication) (enterprise-only)
+- [Managing authentication providers](#managing-authentication-providers)
+- [Configuring authentication providers](#configuring-authentication-providers)
+- [LDAP authentication](#ldap-authentication)
   - [Examples](#ldap-configuration-examples)
   - [Specification](#ldap-specification)
   - [Troubleshooting](#ldap-troubleshooting)
-- [Active Directory authentication](#active-directory-authentication) (enterprise-only)
+- [Active Directory authentication](#active-directory-authentication)
   - [Examples](#active-directory-configuration-examples)
   - [Specification](#active-directory-specification)
   - [Troubleshooting](#active-directory-troubleshooting)
 
 Sensu requires username and password authentication to access the [Sensu dashboard][1], [API][8], and command line tool ([sensuctl][2]).
 For Sensu's [default user credentials][3] and more information about configuring Sensu role based access control, see the [RBAC reference][4] and [guide to creating users][5].
 
-In addition to built-in RBAC, Sensu includes [enterprise-only][6] support for authentication using a Lightweight Directory Access Protocol (LDAP) provider.
+In addition to built-in RBAC, Sensu includes [enterprise-only][6] support for authentication using external authentication providers.
```

Comment:
@palourde What do you think of this sentence? Is "external" ok here? Does Sensu support configuring multiple auth providers for the same instance?

Reply:
Yeah, it makes sense. You can indeed configure multiple providers of the same _type_, e.g. multiple LDAP providers.